### ISSUE TITLE
Add support for links with children in the `RubocopRailsAccessibility/LinkHasHref` rule

### ISSUE THREAD

**Issue by khiga8** [github/rubocop-rails-accessibility#3]
The new [`RubocopRailsAccessibility/LinkHasHref`](https://github.com/github/rubocop-rails-accessibility/blob/main/lib/rubocop/cop/rubocop-rails-accessibility/link_has_href.rb) rule has been extended and still doesn't have support for the following use-case.

```
<%= link_to XXXX do %>
  Text
<% end %>
```

for example:

```
<%= link_to network_dependabot_show_path(update_job_id: update_config.last_full_update_run.id) do %>
  Last checked <%= time_ago_in_words_js Time.at(update_config.last_full_update_run.finished_at.seconds) %>
<% end %>
```
